### ISSUE TITLE
Outdated signtool

### ISSUE THREAD

**Issue by jssuttles** [spatools/node-signtool#2]
Hey,
I noticed that this package hasn't been updated in 7 years.
Is the included signtool out of date?

**Comment by clemp6r** [comment]
Indeed. An update would be welcome in order to support Azure Trusted Signing.
